tests/ballerina-test-integration/src/test/java/org/ballerinalang/test/service/grpc/tool/InvalidServiceContractTestCase.java

```diff
@@ -32,52 +34,60 @@
 /**
  * Class used for test purposes related to invalid input types.
  */
-public class InvalidInputTypes {
+public class InvalidServiceContractTestCase {
+
     private static String protoExeName = "protoc-" + OSDetector.getDetectedClassifier() + ".exe";
     private static Path resourceDir = Paths.get(
-            BalGenToolTest.class.getProtectionDomain().getCodeSource().getLocation().getPath());
-    
+            ClientStubGeneratorTestCase.class.getProtectionDomain().getCodeSource().getLocation().getPath());
+
+    @BeforeClass
+    private void setup() {
+
```

Comment:
Extra new line?

Reply:
Fixed in f2bb0753488e0c357b61ea5a9d3639e43bdf9bfe